### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/advanced/security/oauth2-scopes.md`

### PR DESCRIPTION
**Pull request by ceb10n**

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]

File: docs/pt/docs/advanced/security/oauth2-scopes.md [hunk 1]
```diff
@@ -0,0 +1,786 @@
...
+                * A função de dependência `get_current_user` possui:
+                    * Nenhum escopo necessário.
+                    * Uma dependência utilizando `oauth2_scheme`.
+                    * Um parâmetro `security_scopes` do tipo `SecurityScopes`:
+                        * Este parâmetro `security_scopes` possui uma propriedade `scopes` com uma `list` contendo todos estes escopos declarados acima, então:
+                            * `security_scopes.scopes` terá `["me", "items"]` para a *operação de rota* `read_own_items`.
+                            * `security_scopes.scopes` terá `["me"]` para a *operação de rota* `read_users_me`, porque ela declarou na dependência `get_current_active_user`.
+                            * `security_scopes.scopes` terá `[]` (nada) para a *operação de rota* `read_system_status`, porque ele não declarou nenhum `Security` com `scopes`, e sua dependência, `get_current_user`, não declara nenhum `scopes` também.
+
+/// tip | "Dica"
```
Comment (Joao-Pedro-P-Holanda) [inline docs/pt/docs/advanced/security/oauth2-scopes.md:730]: And here

File: docs/pt/docs/advanced/security/oauth2-scopes.md [hunk 2]
```diff
@@ -0,0 +1,786 @@
...
+
+Porque nós podemos confiar nele para receber o `username` e o `password`, pois nós controlamos isso.
+
+Mas se nós estamos construindo uma aplicação OAuth2 que outros poderiam conectar (i.e., se você está construindo um provedor de autenticação equivalente ao Facebook, Google, GitHub, etc.) você deveria utilizar um dos outros fluxos.
+
+O mais comum é o fluxo implícito.
+
+O mais seguro é o fluxo de código, mas ele é mais complexo para implementar, pois ele necessita mais passos. Como ele é mais complexo, muitos provedores terminam sugerindo o fluxo implícito.
+
+/// note | "Nota"
```
Comment (Joao-Pedro-P-Holanda) [inline docs/pt/docs/advanced/security/oauth2-scopes.md:774]: Finally here, the other places where it appears are on the "Prefer to use the Annotation version if possible"

**Review by Joao-Pedro-P-Holanda — APPROVED** [review]
Great job, just highlighted some minor typos. Seems like the admonition blocks (///) don't need double quotes for the translation. I didn't marked all occurences to avoid polluting the pull request thread.

File: docs/pt/docs/advanced/security/oauth2-scopes.md (hunk 2 above)
Comment (ceb10n) [reply docs/pt/docs/advanced/security/oauth2-scopes.md:774]: I really don't know if the call-outs need or don't need double quotes. But all the translations that I see uses the double quotes for the translated part, as you can see in:

https://github.com/fastapi/fastapi/blob/10f3cb5ab1d5e2848146b85256df056c1bcc9753/docs/pt/docs/environment-variables.md?plain=1#L3

https://github.com/fastapi/fastapi/blob/10f3cb5ab1d5e2848146b85256df056c1bcc9753/docs/pt/docs/virtual-environments.md?plain=1#L5

https://github.com/fastapi/fastapi/blob/10f3cb5ab1d5e2848146b85256df056c1bcc9753/docs/pt/docs/python-types.md?plain=1#L15

etc...

apparently it works with or without quotes, but the standard used in the translated docs is with double quotes.

File: docs/pt/docs/advanced/security/oauth2-scopes.md [hunk 3]
```diff
@@ -0,0 +1,786 @@
...
+
+///
+
+```Python hl_lines="63-66"
+{!> ../../../docs_src/security/tutorial005.py!}
+```
+
+////
+
+Pelo motivo de estarmos declarando estes escopos, eles aparecerão nos documentos da API quando você fizer se autenticar/autorizar.
```
[the remark this answers is not in the document]
Comment (ceb10n) [reply docs/pt/docs/advanced/security/oauth2-scopes.md:205]: You're right, It got a little strange with "quando você fizer se autenticar..."

**Comment by ceb10n** [comment]
> Great job, just highlighted some minor typos. Seems like the admonition blocks (///) don't need double quotes for the translation. I didn't marked all occurences to avoid polluting the pull request thread.

Thank you Joao 😄 Fixed it

**Comment by alejsdev** [comment]
Thanks @ceb10n @nazarepiedady @Joao-Pedro-P-Holanda :rocket:

**Pull request merged**